lib/game.rb

```diff
@@ -100,12 +101,32 @@ def player_sub_placement
       @player_sub = Ship.new("Submarine", 2)
       
       if !@player_board.valid_placement?(@player_sub, @player_sub_coords)
-        p "try again...smh"
+        puts "try again...smh"
       end
 
       break if @player_board.valid_placement?(@player_sub, @player_sub_coords)
     end
     @player_board.place(@player_sub, @player_sub_coords)
   end
 
+  def turn_phase
+    puts "=============COMPUTER BOARD============="
+    @board.render
+    puts "==============PLAYER BOARD=============="
+    @player_board.render(true)
+
+    loop do
+      puts "Enter the coordinate for your shot"
+      player_shot = gets.chomp
+      formatted_player_shot = player_shot.to_s
+
+      if !@board.valid_coordinate?(formatted_player_shot) || @board.cells[formatted_player_shot].fired_upon? 
+        puts "Please enter a valid coordinate"
+      end
+
+      break if @board.valid_coordinate?(formatted_player_shot) && !@board.cells[formatted_player_shot].fired_upon?
+    end #Not receiving any errors, but we will need to test the "re-shoot on a cell logic"
```

Comment:
Will check this out in pry! 

It might be helpful for us to connect via SH to discuss how the computer and player will switch off making their move!